authors/tests/test_tags.py

```diff
@@ -0,0 +1,47 @@
+import json
+
+from rest_framework.views import status
+from django.urls import reverse
+
+# local imports
+from .base_test import TestBase
+
+
+class TagCreation(TestBase):
+    """Test if a tag list is created"""
+
+    def test_tag_addition(self):
+        """Test tags addition to article"""
+        token = self.authentication_token()
+        self.client.credentials(HTTP_AUTHORIZATION='Bearer ' + token)
+        self.client.get(self.get_verify_url(self.user_data))
```

Comment:
Lines 15 to 17 are handled by the `verify_user` method in the TestBase class, please use that instead of repeating them.